centos/6/CentOS-SCLo-scl-rh.repo

```diff
@@ -5,7 +5,7 @@
 
 [centos-sclo-rh]
 name=CentOS-6 - SCLo rh
-baseurl=http://mirror.centos.org/centos/6/sclo/$basearch/rh/
+baseurl=https://vault.centos.org/centos/6/sclo/$basearch/rh/
```

Comment:
I guess using of HTTP makes sense. This distro will not receive updates and so an old ca-certificates package may become a problem for using HTTPS in future.

Reply:
Changed back to HTTP.